core/src/loader.rs

```diff
@@ -1492,7 +1493,123 @@ impl<'gc> Loader<'gc> {
                             &mut activation.context,
                             "complete",
                         );
-                        Avm2::dispatch_event(uc, complete_evt, sound_object);
+                        Avm2::dispatch_event(&mut activation.context, complete_evt, sound_object);
+
+                        let mut call_id3_event = false;
+                        let id3 = activation
+                            .avm2()
+                            .classes()
+                            .id3info
+                            .construct(&mut activation, &[])
+                            .expect("failed to construct ID3Info object");
+                        if let Avm2Object::ScriptObject(id3) = id3 {
+                            let tag = Tag::read_from(response.body.as_slice());
+                            if let Ok(tag) = tag {
+                                call_id3_event = true;
+                                let _ = id3.set_property_local(
+                                    &Multiname::new(
+                                        activation.avm2().find_public_namespace(),
+                                        "album",
+                                    ),
+                                    tag.album().map_or(Avm2Value::Null, |v| {
```

Comment:
Personally I would write this as 
```rust
if let Some(album) = tag.album() {
    id3.set_public_property("name", AvmString::new_utf8(activation.gc(), album).into(),
                            &mut activation).expect("failed set_public_property");
}
```

Reply:
If I use set_public_property function, the public_namespace_vm_internal namespace will be used, is the VM_INTERNAL api version fine?

Reply:
Should be fine, because the property exists already and `public_namespace_vm_internal` matches any namespace.